plutus-contract/src/Plutus/Contract/Test/ContractModel/MissingLovelace.hs

```diff
@@ -0,0 +1,30 @@
+module Plutus.Contract.Test.ContractModel.MissingLovelace
+  ( calculateDelta
+  ) where
+
+import Data.List as List
+import Ledger.Ada qualified as Ada
+import Ledger.Value (Value, noAdaValue)
+import PlutusTx.Prelude qualified as P
+import Wallet.Emulator (Wallet)
+
+calculateDelta :: Value -> Value -> Value -> Wallet -> [(Wallet, [Value])] -> Value
```

Comment:
Missing haddock comments on the function and on the parameters. What's the difference between each of those `Value`s?